doc/contrail_deployment/CAD_for_windows_design.md

```diff
@@ -0,0 +1,50 @@
+# Contrail-ansible-deployer for Windows design document
+This is a short document describing initial support of deploying Windows compute nodes with contrail-ansible-deployer.

```

Comment:
Is it short? Seems pretty medium in length to me ;)

This qualifier is unnecesary